docs/changehistory/NextVersion.md

```diff
@@ -12,6 +12,7 @@ Table of contents:
 - [@itwin/appui-react](#itwinappui-react)
   - [Component changes](#component-changes)
   - [Static manager classes](#static-manager-classes)
+  - [SelectionScopeField](#selectionscopefield)
```

Comment:
Should that be in `Component changes`, or should we have this title level for each components ?